Nodejs/Product/TestAdapter/TestExecutorWorker.cs

```diff
@@ -316,11 +318,47 @@ void RecordEnd(TestCaseResult test, ResultObject resultObject)
             }
         }
 
+        private int GetParentProcessId(uint processId, int counter = 0)
+        {
+            var query = string.Format("SELECT ParentProcessId FROM Win32_Process WHERE ProcessId = {0}", processId);
+            var search = new ManagementObjectSearcher("root\\CIMV2", query);
```

Comment:
I noticed on the documentation that `ManagementObjectSearcher` "cannot be used on by partially trusted code". Is ther a scenario where this might affect us?

Reply:
I looked it up. Partially trusted code is a sandbox like UWP. I think that will never be the case with VS, and also this is going away on net5, so when VS upgrades it will no longer be an issue. ManagementObjectServer is available in net5 and net6, and CAS is not a constraint there.